### ISSUE TITLE
[GLPI 9.4.5] - Dropdowns not being populated except when in debug mode

### ISSUE THREAD

**Issue by b1scuit-thi3f** [glpi-project/glpi#6740]
**Describe the bug**

After upgrading to 9.4.5, dropdowns appear to be missing for all users (including super-admins), as shown below:
![glpi-no-dropdown](https://user-images.githubusercontent.com/14958920/71580152-6cde0600-2af7-11ea-859c-4cfa76bab6cc.png)

However, when in debug mode, the dropdowns are populated:
![glpi-dropdown](https://user-images.githubusercontent.com/14958920/71580167-79625e80-2af7-11ea-9dab-d33aec74336a.png)

**To reproduce**

1. Upgrade to 9.4.5
2. Navigate to assistance and open a ticket
3. Attempt to add a user to the ticket
4. Repeat with debug

**Expected behavior**

Dropdowns should be populated. Data is all still in the database, so I'm guessing something went wrong with a call somewhere?

**Logs**

Operating system: Linux itsm01 4.15.0-1050-aws #52-Ubuntu SMP Thu Sep 12 19:56:00 UTC 2019 x86_64
PHP 7.2.24-0ubuntu0.18.04.1 apache2handler (Core, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, apache2handler, apc,
	apcu, calendar, ctype, curl, date, dom, exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, imap, json, ldap, libxml,
	mbstring, mysqli, mysqlnd, openssl, pcre, pdo_mysql, posix, readline, session, shmop, sockets, sodium, standard, sysvmsg,
	sysvsem, sysvshm, tokenizer, wddx, xml, xmlreader, xmlrpc, xmlwriter, xsl, zlib)
Setup: max_execution_time="600" memory_limit="64M" post_max_size="8M" safe_mode="" session.save_handler="files"
	upload_max_filesize="20M" 
Software: Apache/2.4.29 (Ubuntu) (Apache/2.4.29 (Ubuntu) Server at soc.unipart.com Port 443)
	Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.88 Safari/537.36
Server Software: (Ubuntu)
	Server Version: 5.7.28-0ubuntu0.18.04.4
	Server SQL Mode: 
	Parameters: glpiuser@localhost/glpi
	Host info: Localhost via UNIX socket

Deprecated:  __autoload() is deprecated, use spl_autoload_register() instead in /usr/share/php/CAS/Autoload.php on line 99

mysqli extension is installed
ctype extension is installed
fileinfo extension is installed
json extension is installed
mbstring extension is installed
iconv extension is installed
zlib extension is installed
curl extension is installed
gd extension is installed
simplexml extension is installed
xml extension is installed
ldap extension is installed
imap extension is installed
Zend OPcache extension is installed
APCu extension is installed
xmlrpc extension is installed
CAS extension is installed
exif extension is installed
Database version seems correct (5.7.28) - Perfect!Database version seems correct (5.7.28) - Perfect!
OK/var/log/glpi : OK
OK/etc/glpi/ : OK
OK/var/lib/glpi : OK
OK/var/lib/glpi/_dumps : OK
OK/var/lib/glpi/_sessions : OK
OK/var/lib/glpi/_cron : OK
OK/var/lib/glpi/_graphs : OK
OK/var/lib/glpi/_lock : OK
OK/var/lib/glpi/_plugins : OK
OK/var/lib/glpi/_tmp : OK
OK/var/lib/glpi/_cache : OK
OK/var/lib/glpi/_rss : OK
OK/var/lib/glpi/_uploads : OK
OK/var/lib/glpi/_pictures : OK

**Comment by b1scuit-thi3f** [comment]
Looks like it's a 403 error, as shown below. This only appears when not in debug mode
![image](https://user-images.githubusercontent.com/14958920/71581572-b9c4db00-2afd-11ea-9934-a56a7c274abe.png)

**Comment by b1scuit-thi3f** [comment]
Looks like the error causing this is here (below). This only occurs when debug mode is off.
![image](https://user-images.githubusercontent.com/14958920/71581602-d7924000-2afd-11ea-974d-26753164a356.png)

**Comment by jackburton79** [comment]
Seems to work here (in our testing environment, with the standard interface), is this with the "simplified interface" by chance ?

**Comment by b1scuit-thi3f** [comment]
Hello,
I worked out the issue - it was my bad! Well, some of the permissions on files were wrong. Interestingly, the reason everything work in debug mode was because this calls jquery.fileupload.js. However, when not using debug mode, jquery.fileupload.min.js is called instead. For some reason the permissions issue didn't seem to matter when using the non-minimised file... Perhaps some obscure error there?

Anyway, I've fixed it now as I sorted out the permissions. Whoops
